src/chef-mover/rebar.lock

```diff
@@ -26,7 +26,7 @@
   0},
  {<<"chef_secrets">>,
   {git,"git://github.com/chef/chef_secrets.git",
-       {ref,"58373899212a23e4f37070aaa6d3a751b1281492"}},
+       {branch,"master"}},
```

Comment:
This should point to lbaker/ej fotr now and We can create a ticket to fix expeditor in chef_secrets; once that is done we can update this dep to master.

Reply:
^^ might be for the other erlang deps; lets not update mover unless we have to.

Reply:
NOTE: It appears the lbaker/ej branch has the same expeditor issue.

Reply:
Pointed to chef_secrets lbaker/ej, except for mover.